mobile/src/screens/ChatScreen.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+export const ChatScreen: React.FC = () => {
+  return (
+    <View>
+      <Text>Chat</Text>
+    </View>
+  );
+};
```

Comment:
Super minor thing but can you just add a quick stylesheet so the view is centered in the screen (like the original TODO in App.tsx)? Same with Home and Settings